packages/grafana-data/src/field/fieldDisplay.ts

```diff
@@ -298,3 +293,51 @@ export function getDisplayValueAlignmentFactors(values: FieldDisplay[]): Display
   }
   return info;
 }
+
+function createNoValuesFieldDisplay(options: GetFieldDisplayValuesOptions): FieldDisplay {
```

Comment:
Just out of curiosity why are these functions not tested as units but only as part of testing `getFieldDisplayValues `?

Reply:
Currently, because it is an internal functionality of the `fieldDisplay` module. I would have added tests for this scenario if I could have extracted it as its own module.

It is doable but require some more work/refactorings and I think I am a little bit to new to the code base to make larger changes. It is always good to get some more insight about the code base as a whole before suggesting bigger changes (if that makes sense).

Reply:
Thanks

Reply:
> Currently, because it is an internal functionality of the fieldDisplay module. I would have added tests for this scenario if I could have extracted it as its own module.

Not sure I follow why you would need to extract it to a separate module to unit test it?

Reply:
Hehe, so did my reasoning make sense to you guys? Is it ok to resolve this conversion or should I take any more action on it?

Reply:
no, I think this is fine